src/__tests__/extensions/replay/sessionrecording.test.ts

```diff
@@ -764,9 +744,8 @@ describe('SessionRecording', () => {
             expect(posthog.capture).toHaveBeenCalledWith(
                 '$snapshot',
                 {
-                    $snapshot_bytes: 80,
+                    $snapshot_bytes: 60,
                     $snapshot_data: [
-                        createFullSnapshot(),
```

Comment:
throughout these tests we're no longer manually adding full snapshots so we don't have to assert their presence